sql/core/src/test/scala/org/apache/spark/sql/DatasetBenchmark.scala

```diff
@@ -165,11 +208,23 @@ object DatasetBenchmark {
     val numRows = 100000000
     val numChains = 10
 
-    val benchmark = backToBackMap(spark, numRows, numChains)
+    val benchmark0 = backToBackMapLong(spark, numRows, numChains)
+    val benchmark1 = backToBackMap(spark, numRows, numChains)
     val benchmark2 = backToBackFilter(spark, numRows, numChains)
```

Comment:
`filter()` is handled by `FilterExec()`. Should this PR handle `filter()`, too? Or, do I open another PR for `filter()`?